index.html

```diff
@@ -10,46 +10,236 @@
+     <ul class="resources" title="resources">
+      <li><a  class="button" href="#">HTML</a> <li>
+      <li><a  class="button" href="#">CSS</a> </li>
+      <li><a  class="button" href="#">javaScript</a></li>      
+    </ul>
+
+    <button onclick="togglePopup2()"  class="normalbutton2 seeproject">See Project</button>
+    
+    </div>
+  </div>
+
+  <!--page 4--> 
+  
+  <div class="card1" title="works-card-3">
+    <img class="avator1" id="avator" src="sources/Portfolio3.png" alt="Avatar">
+    <div class="container">
+      <h2><b>Tonic</b></h2>
+
+     <ul class="tonic">
+      <li><a class="tonic canopy" href="#">CANOPY</a></li>
+      <li class="tonic dot"></li>
+      <li class="tonic back">Back End Dev</li>
+       <li class="tonic dot"></li>
+       <li class="tonic year">2015</li>
+     </ul>
 
-   <div class="smedia" title="social-media-accounts">
+     <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
    
-   <a class="LetsConnect" href="#">LET’S CONNECT </a>
+     <ul class="resources" title="resources">
+      <li><a  class="button" href="#">HTML</a> <li>
+      <li><a  class="button" href="#">CSS</a> </li>
+      <li><a  class="button" href="#">javaScript</a></li>      
+    </ul>
+
+    <button onclick="togglePopup3()"  class="normalbutton seeproject">See Project</button>
+    
+    </div>
+  </div>
+
+ <!-- page 5-->
+  <div class="card2" title="works-card-4">
+    <img class="avator4" id="avator" src="sources/Portfolio4.png" alt="Avatar">
+    <div class="container2">
+      <h2><b>Multi-Post</br>
+        Stories</b></h2>
+
+     <ul class="tonic">
+      <li><a class="tonic canopy" href="#">CANOPY</a></li>
+      <li class="tonic dot"></li>
+      <li class="tonic back">Back End Dev</li>
+       <li class="tonic dot"></li>
+       <li class="tonic year">2015</li>
+     </ul>
+
+     <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
    
-   <ul class="topicons">
-     <li><a href="#"> <img src="sources/Vector4.png" alt="twitter"></a></li>    
-     <li><a href="#"> <img src="sources/linkedin.png" alt="linkedin"></a></li>     
-     <li><a href="#"> <img src="sources/Vector5.png" alt="medium"></a></li>        
-     <li><a href="#"> <img src="sources/Vector1.png" alt="github"></a></li>        
-     <li><a href="#"> <img src="sources/Vector3.png" alt="chill"></a></li>   
+     <ul class="resources" title="resources">
+      <li><a  class="button" href="#">HTML</a> <li>
+      <li><a  class="button" href="#">CSS</a> </li>
+      <li><a  class="button" href="#">javaScript</a></li>      
+    </ul>
+
+    <button onclick="togglePopup4()" class="normalbutton2 seeproject">See Project</button> 
+     
+    </div>
+  </div>
+</section>
+
+<!--Page 6-->
+<article class="aboutme" id="about" title="about-me">
+
+<section class="top">
```

Comment:
Your profile overall does not seem to have a background.
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219085755-8c3e8b19-1bc7-4734-9e57-cb545ff6343d.png)  | ![image](https://user-images.githubusercontent.com/61361037/219085550-a5ade6aa-b14d-4493-b8ae-b59da962787c.png)  |

In this Figma Template, the About section, for example, is in a form of a card background with a rounded top-right border. Kindly look at the design again